### ISSUE TITLE
`waitFor` method

### ISSUE THREAD

**Issue by icebob** [ice-services/moleculer#87]
Create a `waitFor(serviceName: string|Array<string>): Promise` method in `Service` class.
With it you can wait for a service or multiple services to be available.

For example:

```js
module.exports = {
    name: "seed",
    
    started() {
        this.seedDB();
    },

    methods: {
        seedDB() {
            this.waitFor(["posts", "users"]).then(() => {
                //... Seed posts & users collections
            });
        }
    }
};
```

**Comment by WoLfulus** [comment]
Awesome! :D

**Comment by icebob** [comment]
It will be released in v0.11
